packages/contracts/.cct/setup-fork.js

```diff
@@ -66,6 +74,15 @@ export const setupFork = async (
     await testToken1Contract.mint(signer.address, utils.parseUnits("100", 18));
     await testToken2Contract.mint(signer.address, utils.parseUnits("100", 18));
 
+    // give us some wrapped native currency too
+    await new Contract(
```

Comment:
I don't understand why this was added, here we have the TST1 and TST2 tokens.

Reply:
Yea it was just to have a native currency, if we don't need it, we can remove it.